### PR TITLE
python39: update to 3.9.18

### PR DESCRIPTION
**Pull request by Bill-Sommerfeld**

### CODE REVIEW

File: components/python/python39/Makefile
Comment (AndWac) [inline components/python/python39/Makefile:114]: 1. When you change a package you either need to change COMPONENT_VERSION or increment/set COMPONENT_REVISION.
2. Our python builds receipts are broken for some time. We cannot publish updates before this has been fixed.

Comment (Bill-Sommerfeld) [reply components/python/python39/Makefile:114]: > Our python builds receipts are broken for some time. We cannot publish updates before this has been fixed.

I'd be happy to help with that if possible if I can get a brain dump on what needs to happen (I'm also interested in helping bring newer python versions into OI)

Comment (AndWac) [reply components/python/python39/Makefile:114]: When you run gmake sample-manifest you'll find out that the newly generated one is missing the tk library. I haven't found out yet why this doesn't get build anymore.

Comment (Bill-Sommerfeld) [reply components/python/python39/Makefile:114]: I think I found it - setup.py isn't looking in /usr/lib/64 for libtcl and libtk which is the only place you'll find it if there aren't 32-bit libraries.

Comment (mtelka) [reply components/python/python39/Makefile:114]: > I'm also interested in helping bring newer python versions into OI

Such help is always welcome.  Python 3.12 would be great.  Thanks.

Comment (mtelka) [reply components/python/python39/Makefile:114]: @AndWac, FYI, addition of new major Python version needs special handling to make sure it does not break rest of oi-userland build framework or user's machines, so please wait for my review/approval when such PR appears.

@Bill-Sommerfeld, do not worry about the above :-).

Comment (AndWac) [reply components/python/python39/Makefile:114]: Sure

**Review by mtelka — APPROVED** [review]

**Pull request merged**
